### ISSUE TITLE
[backstage] scaffolder templates

### ISSUE THREAD

**Issue by phac008** [suxess-it/sx-cnp-oss#682]
currently all scaffolder templates are kind of specific for our different showcases. 
there should be a general approach to get rid of hardcoded

github urls 
certificate annotations

may a template for generating backstage templates would be a nice solution to get this things done in a fancy way...
tbd

**Comment by phac008** [comment]
put back this topic in timeline

**Comment by phac008** [comment]
the separation between prime and oss should also be taken into account accordingly in the templates. 
e.g. team-onboarding target repo should include cluster name somehow

**Comment by jkleinlercher** [comment]
I guess our current approach of templating the scaffolder template solves this issue? @phac008 ?

**Comment by phac008** [comment]
yes, approach solves topic - cleanup needed for some of the demo templates, but issue is solved
